test/account.spec.ts

```diff
@@ -13,8 +14,7 @@ const {
   isValidChecksumAddress,
   isValidAddress,
   toChecksumAddress,
-} = require('../src')
-import BN = require('bn.js')
+} from '../src'
```

Comment:
Some unrelated import improvements, nice, and justifiable since only touching the tests.